jobs/acceptance-tests/templates/config.json.erb

```diff
@@ -0,0 +1,48 @@
+<%
+require 'json'
+
+config = p('acceptance_tests')
+config[:use_existing_user] = !properties.acceptance_tests.existing_user.nil?
+config[:keep_user_at_suite_end] = !properties.acceptance_tests.existing_user.nil?
+config[:use_existing_organization] = !properties.acceptance_tests.existing_organization.nil?
+config[:artifacts_directory] ||= '/var/vcap/sys/log/acceptance_tests/'
+
+# TODO: do something with p('acceptance_tests.include')
```

Comment:
Is this TODO still relevant? We are doing _something_ with it below.

Reply:
Fixed (comment removed) in 294f869e844409006b8de0f79291e1e904ecb3ed